### ISSUE TITLE
Hero featured content

### ISSUE THREAD

**Issue by hannahkane** [mozilla/network#291]
quick design exploration to inform discussion

Estimate is design only, because ticket is design only for now.

**Comment by xmatthewx** [comment]
Here's a crappy wireframe to clarify. 

Include the people box in the mockup. This will surface conflicts in hierarchy and clarity. Feel free to use v2 of Tais' latest iteration. 

![hero-content](https://cloud.githubusercontent.com/assets/53411/24977870/a502b49a-1f9c-11e7-803d-6c8820f9842c.jpg)

**Comment by sabrinang** [comment]
Here is an iteration:
![hero-featured-popup](https://cloud.githubusercontent.com/assets/2492510/24984522/be10a252-1fa2-11e7-9e47-aedf93656e4d.png)

These look like weird popups and there are a lot of competing buttons going on. I would focus on featuring people OR internet health and not both. Also, the internet health section does not have any context after people read the headline/subhead of the site. I think this may confuse people with the amount of choice on this viewport and takes away from members.

**Comment by xmatthewx** [comment]
Despite your best effort @sabrinang, we are going to kill this feature. 😢 😢 😭 😭 

We did discover the underlying problem: How do visitors learn more about a _healthy internet_ and what that term means? The answer could be a text link after "We are a global network..." that says "Learn about Internet Health." Or, perhaps we need a simple bar below the people feature that says: "What's a Healthy? Learn more..."

Closing this. Will file that.
